### PR TITLE
Fixed image attachment e2e cypress test

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Fixes the failing tests in `image_attachmemt_spec.js`.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-52523

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by harshilsharma63** [comment]
The concerned tests are passing now-
Screenshot 2023-05-18 at 3 56 32 PM

File: e2e-tests/cypress/tests/integration/channels/messaging/image_attachment_spec.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ describe('Image attachment', () => {
         // # File thumbnail should have correct dimensions
         verifyFileThumbnail({
             filename,
-            actualImage: {height: 151, width: 955},
+            actualImage: {height: 144, width: 906},
```
Comment (larkox) [inline e2e-tests/cypress/tests/integration/channels/messaging/image_attachment_spec.js:100]: Does it make sense to have this in some constant that is imported here? Maybe not. 0/5

**Review by larkox — APPROVED** [review]
LGTM. Just one nit for future though.

**Pull request merged**